Update root and todo readme files. (#67)
dgraph-io/graphql-sample-apps: todo-app-react/README.md

<div style="padding-top: 5px; padding-bottom: 10px;">
  <h1 align="center">ToDo React App</h1>
  <h2 align="center">
    A One-Click Deployable App with<br />
    <a href="https://dgraph.io/slash-graphql" target="_blank">
      <img src="https://dgraph.io/assets/images/slashgraphql-logo.svg" alt="Slash GraphQL" />
    </a><br />

>     A fully managed GraphQL backend service

Reviewer: ```suggestion
    A fully-managed GraphQL backend service
```

  </h2>
</div>

<h3 align="center"><a href="https://slash.dgraph.io/_/one-click?app=todo" target="_blank">Deploy Now</a> for free!</h3>

> To-Do is a sample app that lets uers manage the tasks on their personal to-do list. Developers can see how to use react hooks with apollo client to easily create, read, update, and delete To-Do items. Deploying this app on Slash deploys both the back-end database service and a front-end react app in a single click, no credit card required. To learn more about this sample app, see: [Building a To-Do List React App with Dgraph](https://dgraph.io/blog/post/building-todo-list-react-dgraph/)

Reviewer: ```suggestion
To-Do is a sample app that lets users manage the tasks on their personal to-do list. This app demonstrates how to use React hooks with an Apollo client to easily create, read, update, and delete to-do list items. Deploying this app on Slash GraphQL deploys both the back-end database service and a front-end React app in a single click, no credit card required. To learn more about this sample app, see: [Building a To-Do List React App with Dgraph](https://dgraph.io/blog/post/building-todo-list-react-dgraph/)
```
